### DISCUSSION TITLE
School not listed

### DISCUSSION THREAD

**Discussion by ssdrive-flaktra** [community/community#55404] [GitHub Education] [answered]
### Hi everyone,

Hi! My school is not listed in the list of schools in the text box and I am require to enter a school ID when I already have a school email linked? Our school doesn't have IDs so not really sure what to do.

**Comment by ryanlua** [comment, marked as answer]
Hi @AToska21,

You do not need a Student ID to get the GitHub Student Developer Pack, for more information see "[How to get the GitHub Student Developer Pack without a student ID](https://github.blog/2019-07-30-how-to-get-the-github-student-developer-pack-without-a-student-id/)"

As for your school, you can have an administrator sign up by making them fill out the [GitHub Campus Program Application](https://education.github.com/partner_school_applications/apply).
